2.14.0/linux/alpine/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM alpine
+ENV DDF_VERSION=2.14.0-SNAPSHOT
+RUN apk add --no-cache curl unzip
+RUN curl -o ddf.zip \
+  "http://artifacts.codice.org/service/local/artifact/maven/content?g=org.codice.ddf&a=ddf&v=$DDF_VERSION&r=snapshots&e=zip"
+RUN unzip ddf.zip
+RUN mv ddf-${DDF_VERSION} /ddf
+
+FROM codice/ddf-base:2.12-alpine
```

Comment:
❓ Do we not create a new version of ddf-base and redeploy for every new ddf version?

Reply:
nope, i've only done it when something significant had to change for automating a specific version and there hasn't been anything breaking like that for a while. Sometimes I did it just to be safe, at some point I intend to remove the need for multiple versions of the scripts as well